### PR TITLE
Feature/encourage prisoners to explore parts of the hub

### PR DESCRIPTION
**Pull request by BarryGeeMOJ**
### Context

> Does this issue have a Trello card?
https://trello.com/c/qOyx9cQu/921-encourage-prisoners-to-explore-parts-of-the-hub

> If this is an issue, do we have steps to reproduce?
n/a

### Intent

> What changes are introduced by this PR that correspond to the above card?
- Added new explore content block, query and tests

> Would this PR benefit from screenshots?
Screenshot 2022-06-09 at 13 15 14

### Considerations

> Is there any additional information that would help when reviewing this PR?
n/a

> Are there any steps required when merging/deploying this PR?
n/a

### Checklist

- [ ] This PR contains **only** changes related to the above card
- [x] Tests have been added/updated to cover the change
- [ ] Documentation has been updated where appropriate
- [x] Tested in Development

### CODE REVIEW

File: server/views/pages/home-new.html [hunk 1]
```diff
@@ -30,6 +30,13 @@
     id: 'recentlyAdded',
     data: recentlyAddedContent
   }) }}
+
+  {{ hubContentBlock({
+    title: 'Explore to Hub',
```
Comment (leonk) [inline server/views/pages/home-new.html:35]: Typo? Think this should be "Explore the Hub"

File: assets/sass/components/_home-content.scss [hunk 2]
```diff
@@ -249,6 +249,6 @@
 
 .hub-content-block__title-bar {
   display: flex;
-  align-items: flex-start;
-  justify-content: space-between;
+  align-items: center;
+  justify-content: flex-start;
 }
```
Comment (leonk) [inline assets/sass/components/_home-content.scss:254]: Have these changes been checked with the current homepage? (As it looks like they might effect it).

**Review by leonk — CHANGES_REQUESTED** [review]

File: assets/sass/components/_home-content.scss (hunk 2 above)
Comment (BarryGeeMOJ) [reply assets/sass/components/_home-content.scss:254]: the hub-content-block is only used on the new homepage

File: server/views/pages/home-new.html (hunk 1 above)
Comment (BarryGeeMOJ) [reply server/views/pages/home-new.html]: updated.

File: server/repositories/cmsQueries/exploreContentQuery.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
+const { DrupalJsonApiParams: Query } = require('drupal-jsonapi-params');
+const { getSmallTile } = require('../../utils/jsonApi');
+
+class ExploreContentQuery {
+  static #TILE_FIELDS = [
+    'drupal_internal__nid',
+    'title',
+    'field_moj_thumbnail_image',
+    'field_thumbnail_image',
```
Comment (jonhindle) [inline server/repositories/cmsQueries/exploreContentQuery.js:9]: Hasn't `field_thumbnail_image` been removed?

Comment (jonhindle) [reply server/repositories/cmsQueries/exploreContentQuery.js:9]: Looks like it needs removing from `server/repositories/cmsQueries/recentlyAddedContentQuery.js` and probably `server/repositories/cmsQueries/homePageQuery.js`. Also there specs 🤔

File: server/services/__tests__/cmsService.spec.js [hunk 4]
```diff
@@ -973,4 +976,36 @@ describe('cms Service', () => {
       expect(result).toBe(resObject);
     });
   });
+
+  /*
+    to do - explore
+  */
```
Comment (jonhindle) [inline server/services/__tests__/cmsService.spec.js:982]: Can these comments be removed now?

File: server/repositories/cmsQueries/exploreContentQuery.js (hunk 3 above)
Comment (BarryGeeMOJ) [reply server/repositories/cmsQueries/exploreContentQuery.js:9]: updated

File: server/services/__tests__/cmsService.spec.js (hunk 4 above)
Comment (BarryGeeMOJ) [reply server/services/__tests__/cmsService.spec.js:982]: updated

File: server/services/__tests__/cmsService.spec.js [hunk 5]
```diff
@@ -973,4 +976,33 @@ describe('cms Service', () => {
...
+      );
+    });
+
+    it('should return a result when cmsApi.get is called', async () => {
+      expect(result).toBe(resObject);
+    });
+  });
+  /*
+    end - to do - explore
+  */
```
Comment (jonhindle) [inline server/services/__tests__/cmsService.spec.js:1007]: Remove comment?

Comment (BarryGeeMOJ) [reply server/services/__tests__/cmsService.spec.js]: not sure how i missed that! - updated.

**Review by jonhindle — APPROVED** [review]
lgtm

**Pull request merged**